### ISSUE TITLE
Strong emphasis identified in HTML

### ISSUE THREAD

**Issue by treyhunner** [rvben/rumdl#41]
This file:

```md
# Test

This is some inline HTML:

<iframe width="100%" height="550" frameborder="0" src="https://pythontutor.com/iframe-embed.html#code=class%20Point%3A%0A%20%20%20%20def%20__init__%28self,%20x,%20y,%20z%29%3A%0A%20%20%20%20%20%20%20%20self.x%20%3D%20x%0A%20%20%20%20%20%20%20%20self.y%20%3D%20y%0A%20%20%20%20%20%20%20%20self.z%20%3D%20z%0A%20%20%20%20%20%20%20%20%0A%20%20%20%20def%20__repr__%28self%29%3A%0A%20%20%20%20%20%20%20%20return%20f%22Point%28%7Bself.x%7D,%20%7Bself.y%7D,%20%7Bself.z%7D%29%22%0A%0Ap%20%3D%20Point%281,%202,%203%29%0Ap.x%20%3D%2040&codeDivHeight=400&codeDivWidth=350&cumulative=false&curInstr=7&heapPrimitives=nevernest&origin=opt-frontend.js&py=3&rawInputLstJSON=%5B%5D&textReferences=false"> </iframe>

And this is **bolded text**.
```

Shows this incorrect error (due to the `__repr__` in the filename:

```bash
$ uv tool run rumdl check --disable MD033 file12.md
file12.md:7:13: [MD050] Strong emphasis should use __ instead of ** [*]

Issues: Found 1 issues in 1 file (37ms)
Run with `--fix` to automatically fix 1 of the 1 issues
```

**Comment by rvben** [comment]
✅ **Fixed in v0.0.113**

MD050 now properly skips content inside HTML tags and won't flag strong emphasis patterns in HTML attributes like iframe src URLs.

The fix adds HTML tag detection to all MD050 strong emphasis checking.

Thanks for the great example\!
